osu.Game.Tests.iOS/osu.Game.Tests.iOS.csproj

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\NUnit.3.11.0\build\NUnit.props" Condition="Exists('..\packages\NUnit.3.11.0\build\NUnit.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">iPhoneSimulator</Platform>
+    <OutputType>Exe</OutputType>
+    <ProjectGuid>{65FF8E19-6934-469B-B690-23C6D6E56A17}</ProjectGuid>
+    <RootNamespace>osu.Game.Tests</RootNamespace>
+    <AssemblyName>osu.Game.Tests.iOS</AssemblyName>
+  </PropertyGroup>
+  <Import Project="..\osu.iOS.props" />
+  <ItemGroup>
+    <None Include="Info.plist" />
+    <None Include="Entitlements.plist" />
+    <None Include="..\osu.iOS\libbass.a">
+      <Link>libbass.a</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="..\osu.iOS\libbass_fx.a">
+      <Link>libbass_fx.a</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <LinkDescription Include="..\osu.iOS\Linker.xml">
+      <Link>Linker.xml</Link>
+    </LinkDescription>
+    <Compile Include="Application.cs" />
+    <Compile Include="AppDelegate.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\osu.Game.Tests\**\Beatmaps\**\*.cs">
```

Comment:
is this because of some incompatible tests?

Reply:
For the unit tests to be discovered, they need to be in the executing assembly, it seems.  This means I couldn't split the tests into common, desktop and iOS projects.  Instead, I made an iOS project that directly links to the source and compiles with the same assembly name.

Reply:
I think it's probably best to not do this, as it's 100% going to get forgotten when we add a new folder or namespace.

Reply:
I wasn't particularly happy with it either.  Another option I thought of is to include `*.cs` across the entire project, then exclude anything it puts in the build directory.